src/test/resources/sql/scheduledPrisonerReturnFromCourt_clean.sql

```diff
@@ -0,0 +1,7 @@
+DELETE from COURT_EVENTS where EVENT_ID in (455654698,455654697);
```

Comment:
Use the offender_book_id of the offender_bookings row you created for these deletes.

Reply:
done